### DISCUSSION TITLE
IOS app stuck?

### DISCUSSION THREAD

**Discussion by danconde** [azadbolour/eagerwords-community#15] [Q&A] [answered]
The game (test edition) got stuck.  Only button I can press was Done. There were no other indications that the game was over, or that both sides had no moves to make.  I could only press 'Done' so the attached log may not reflect what was seen at prior game in case the log go reset.  I did not try to terminate the app and chose to press 'Done'
[game.log](https://github.com/azadbolour/eagerwords-community/files/11279516/game.log)

**Comment by azadbolour** [comment, marked as answer]
The attached log file shows your last game action as posting a word that was not found in the dictionary. The code indicates that the 'Play' button becomes enabled after the error is reported, unless the game is finished or the player is timed out. The game could be finished by a subsequent bot play (after the grace period). The player is timed out if his clock runs down. I am not sure if either of these conditions held in your case. I am unable to reproduce this issue. If it crops up again and is unrelated to a finished game or a timed out user, please let me know, and I will have to consider including additional logging to pinpoint the problem.

Azad
